File: C++/BASICS/DIFF.cpp
#include<iostream>
using namespace std;
int main()
{
    int a,b,c;
    cout<<"Enter 1st no...";
    cin>>a;
    cout<<"Enter 2nd no...";
    cin>>b;
    if(a>b)
    {
        c=a-b;
    }
    else
    {
        c=b-a;
    }
    cout<<"Diff is "<<c<<".";
}

File: JAVA/BASICS/DIFF.java
import java.util.*;
public class DIFF {
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int a,b,c;
        System.out.println("Enter 1st no...");
        a=sc.nextInt();
        System.out.println("Enter 2nd no...");
        b=sc.nextInt();
        if(a>b)
        {
            c=a-b;
        }
        else
        {
            c=b-a;
        }
        System.out.println("Diff is "+c+".");
    }
    
}
